frontend/openchat-agent/src/services/user/user.client.ts

```diff
@@ -123,6 +127,28 @@ export class UserClient extends CandidService implements IUserClient {
         );
     }
 
+    @profile("userClient")
+    getInitialStateV2(): Promise<InitialStateV2Response> {
+        const args = {
+            disable_cache: apiOptional(identity, false),
```

Comment:
this just always returns false

Reply:
Yeah previously it was driven off a local storage setting but that isn't accessible in the agent so I've just hardcoded `false` for now.